File: LeetCode/0125. Valid Palindrome/C#/01. Valid Palindrome.cs
using System.Text.RegularExpressions;

bool IsPalindrome(string s)
{
    Regex regex = new Regex("[^0-9A-Za-z]");
    string clrBaseText = regex.Replace(s, string.Empty).ToLower();
    string clrReverseBaseText = new string(clrBaseText.ToCharArray().Reverse().ToArray());
    return clrBaseText == clrReverseBaseText;
}

File: LeetCode/0125. Valid Palindrome/JavaScript/01. Valid Palindrome.js
var isPalindrome = function(s) {
    const regexPattern = new RegExp("[^0-9A-Za-z]", "g");
    let clrBaseText = s.replace(regexPattern, "").toLowerCase();
    let clrReverseBaseText = clrBaseText.split("").reverse().join("");
    return clrBaseText == clrReverseBaseText
};